recovery/elliptics_recovery/dc_recovery.py

```diff
@@ -587,6 +591,16 @@ def recover(ctx):
                          .format(options.batch_size, repr(e), traceback.format_exc()))
     log.info("Using batch_size: {0}".format(ctx.batch_size))
 
+    try:
+        ctx.pool_size = int(options.pool_size)
+        if ctx.pool_size <= 0:
+            raise ValueError("Pool size should be positive: {0}"
+                             .format(ctx.pool_size))
+    except Exception as e:
+        raise ValueError("Can't parse poolsize: '{0}': {1}, traceback: {2}"
+                         .format(options.pool_size, repr(e), traceback.format_exc()))
+    log.info("Using pool_size: {0}".format(ctx.pool_size))
```

Comment:
Use native formatter instead of `format` with logs
```suggestion
    log.info("Using pool_size: %s", ctx.pool_size)
```